### ISSUE TITLE
CREEL_FISH duplicate fish

### ISSUE THREAD

**Issue by kaijagahm** [MFEh2o/db#68]
`cf <- dbTable("CREEL_FISH")`

`cf %>% group_by(fishID) %>% filter(n() > 1) %>% as.data.frame() # see any duplicate fishID's`

```
  projectID                                                             creelID
1        36 BK_BoatLaunch_20180628_1330_CR_fishscapesCreel.Interview.20180910_2
2        36 BK_BoatLaunch_20180628_1330_CR_fishscapesCreel.Interview.20180910_2
                                                                 fishID creelNumber fishNumber species
1 BK_BoatLaunch_20180628_1330_CR_fishscapesCreel.Interview.20180910_2_1           2          1 walleye
2 BK_BoatLaunch_20180628_1330_CR_fishscapesCreel.Interview.20180910_2_1           2          1 walleye
  speciesCode_wiDNR fishLength notes enteredBy                     updateID
1               X23        424  <NA>        AT originalCreelUpdate_20190125
2               X24        482  <NA>        AT originalCreelUpdate_20190125
```
`
Different fish length, and different species codes even though the species name is the same. Maybe start by checking what the correct code is for walleye, then check data sheets to see if this is an actual duplicate or if it should be two separate fish.

**Comment by kaijagahm** [comment]
Double check with Ashley: are the lengths in the spreadsheet in cm (which would correspond to mm in the database)

**Comment by kaijagahm** [comment]
As per Ashley's email, the first fish listed here is fish 2 from the BK creel 2 on 6/28/2018.

The second fish is fish 1 from MS 7/9/2018 (creel 2).

Both should have X22 as their species code.

**Comment by kaijagahm** [comment]
Closed with db version 3.5.0, 9/30/2020
